gradle/libs.versions.toml

```diff
@@ -36,17 +51,28 @@ firebaseFirestore = "25.1.0"
 firebaseUiAuth = "8.0.0"
 firebaseFirestoreKtx = "25.1.0"
 espressoIntents = "3.6.1"
+
 navigationRuntimeKtx = "2.8.2"
 navigationCommonKtx = "2.8.2"
 navigationTesting = "2.8.2"
 
+
 # OSM Librairies
 osmdroid = "6.1.20"
 okhttp = "4.12.0"
 
 [libraries]
 androidx-core-ktx = { group = "androidx.core", name = "core-ktx", version.ref = "coreKtx" }
+androidx-credentials = { module = "androidx.credentials:credentials", version.ref = "credentials" }
+androidx-credentials-play-services-auth = { module = "androidx.credentials:credentials-play-services-auth", version.ref = "credentials" }
+androidx-credentials-play-services-auth-v10 = { module = "androidx.credentials:credentials-play-services-auth", version.ref = "credentialsPlayServicesAuthVersion" }
+androidx-credentials-play-services-auth-v150alpha05 = { module = "androidx.credentials:credentials-play-services-auth", version.ref = "credentialsPlayServicesAuth" }
+androidx-credentials-v150alpha05 = { module = "androidx.credentials:credentials", version.ref = "credentialsVersion" }
+androidx-credentials-vlatestversion = { module = "androidx.credentials:credentials", version.ref = "androidxCredentials" }
```

Comment:
Question: Why are there three declarations of each of the two dependencies declared here?

Reply:
same as the comment above, it should be better now